src/python/pants/vcs/git_test.py

```diff
@@ -20,7 +20,8 @@
 
 
 def init_repo(remote_name: str, remote: PurePath) -> None:
-    subprocess.check_call(["git", "init", "--initial-branch=main"])
```

Comment:
Didn't have a new enough `git`.

Reply:
.. and won't/can't upgrade?? 🤪

Reply:
I mean I could, but seems restrictive to not be tolerant of git versions for devs 🤷‍♂️

Reply:
🤷🏽

Reply:
Yeah seems fine to drop